.github/workflows/python-package.yml

```
@@ -20,11 +20,14 @@ jobs:
       wheel-path: ${{ steps.get-darker-version.outputs.wheel-path }}
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
```

Is there any reason to not specify `python-version` here?
```suggestion
      - uses: actions/setup-python@v5
        with:
          python-version: '3.x' 
```

---

> Is there any reason to not specify `python-version` here?

Probably because `uv build` doesn't try to install anything in the system Python, so Ubuntu's system Python protection doesn't kick in?